OPHD/States/MapViewStateDraw.cpp

```diff
@@ -170,11 +171,16 @@ void MapViewState::drawResourceInfo()
 	}
 
 	// Capacity (Storage, Food, Energy)
+	int foodStorage = 0;
+	for (auto& foodProduction : enumerateComponent<FoodProduction>())
+	{
+		foodStorage += foodProduction.foodCapacity();
+	}
```

Comment:
The prior method of counting food storage capacity did not take Command Center capacity into account. Now actually counting the capacity, so that every food production structure can have an arbitrary capacity.